### PR TITLE
underwriter signed quote endpoint

### PR DESCRIPTION
**Pull request by FredrikAreschoug**

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/campaign/integration/underwriter/dto/QuoteDto.kt [hunk 1]
```diff
@@ -51,6 +55,7 @@ sealed class QuoteData {
     val street: String? = null,
     val city: String? = null,
     val zipCode: String? = null,
+    val floor: Int? = null,
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/campaign/integration/underwriter/dto/QuoteDto.kt]: This is never used

File: src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java [hunk 2]
```diff
@@ -21,4 +20,56 @@
     private final LocalDateTime activeFrom;
     private final Boolean isFromPartner;
 
+    public SingedContractCommand(
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java]: Singed is a good champion in League of Legends

File: src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java [hunk 3]
```diff
@@ -7,7 +7,6 @@
 import java.time.Instant;
 import java.time.LocalDateTime;
 
-@Value
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java]: You can just make the variables public so you don't need to to delombok the whole file

File: src/main/java/com/hedvig/productPricing/service/serviceIntegration/botService/dto/Address.java [hunk 4]
```diff
@@ -9,15 +9,22 @@
...
   private String zipCode = "";
   private Integer floor = 0;
 
+  public Address() {}
+
+  public Address(String street, String city, String zipCode, Integer floor) {
+     this.street = street;
+     this.city = city;
+     this.zipCode = zipCode;
+     this.floor = floor;
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/serviceIntegration/botService/dto/Address.java]: floor is never used

File: src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java (hunk 2 above)
Comment (meletios) [reply src/main/java/com/hedvig/productPricing/service/commands/SingedContractCommand.java]: Really good  tanky player, i would add.

**Pull request merged**